app/src/main/java/com/simplecity/amp_library/utils/MusicUtils.java

```diff
@@ -30,35 +30,35 @@
         int NEW_PLAYLIST = 2;
     }
 
+    /**
+     * Passes along a list of songs, wrapped in a Single obj, to be played, starting at position 0.
+     */
     public static void playAll(Single<List<Song>> songsSingle, UnsafeConsumer<String> onEmpty) {
         songsSingle.observeOn(AndroidSchedulers.mainThread())
-                .subscribe(songs -> playAll(songs, onEmpty));
```

Comment:
This feels like a subtle _looking_ but intentional change which actually changes Shuttle's default queuing/shuffling behaviour (for the better) - shuffle will now turn off when selecting a new list of songs to play, is that correct?

In other words, if you have shuffle on, and then you hit 'play' on an album, shuffle will be turned off.

I agree with this change (something I've been meaning to do for ages). Is this something that was required for album-shuffle, or was it just an aside / something else you wanted to fix? (Just curious!)

Reply:
It was just an aside, I thought it made more sense working that way.